data/authors.yaml

```diff
@@ -99,4 +99,6 @@ authors:
       name: Prosper Otemuyiwa
       bio: Staff Developer Advocate, Sourcegraph
       twitterLink: https://twitter.com/unicodeveloper
-      linkedInLink: https://www.linkedin.com/in/prosperotemuyiwa/
\ No newline at end of file
+      linkedInLink: https://www.linkedin.com/in/prosperotemuyiwa/
+    - slug: anja-bender
+      name: Anja Bender
```

Comment:
@Lumary2 — if you would like to add a bio, or any links, feel free to add them to this yaml file in a new PR! Thanks 😄